app/controllers/concerns/hydranorth/files_controller_behavior.rb

```diff
@@ -47,18 +52,9 @@ def edit_form
 
       @community_collections = collections_for_community(@generic_file.belongsToCommunity.first)
 
-      if @generic_file[:resource_type].include? Sufia.config.special_types['cstr']
-        Hydranorth::Forms::CstrEditForm.new(@generic_file)
-      elsif @generic_file[:resource_type].include? Sufia.config.special_types['ser']
-        Hydranorth::Forms::SerEditForm.new(@generic_file)
-      elsif @generic_file[:resource_type].include? Sufia.config.special_types['thesis']
-        Hydranorth::Forms::ThesisEditForm.new(@generic_file)
-      else
-        Hydranorth::Forms::GenericFileEditForm.new(@generic_file)
-      end
+      edit_form_class.new(@generic_file)
```

Comment:
ditto this https://github.com/projecthydra/sufia/blob/6.x-stable/app/controllers/concerns/sufia/files_controller_behavior.rb#L179